website/content/en/preview/troubleshooting.md

```diff
@@ -6,14 +6,39 @@ description: >
   Troubleshoot Karpenter problems
 ---
 
+## Controller
+
+### Enable debug logging
```

Comment:
Can we include info on how to do this with helm as well?

Reply:
Do you know what the differences is between `logLevel` which is used in the zap-logger-config and `controller.logLevel` which is used separately in the helm template?

https://github.com/aws/karpenter/blob/main/charts/karpenter/templates/configmap-logging.yaml#L39

Reply:
`controller.logLevel` is an artifact of an old release and needs to be pulled out and removed from the values.yaml. I'll file a separate issue to cleanup the helm values